pkg/webhook/stack/validatingwebhook.go

```diff
@@ -71,6 +73,13 @@ func (v *stackValidator) validateStackFn(ctx context.Context, stack *kabanerov1a
 			err = fmt.Errorf(reason)
 			return false, reason, err
 		}
+
+		_, err := semver.Parse(version.Version)
+		if err != nil {
+			reason = fmt.Sprintf("Stack %v %v spec.Versions[].Version must be semver. %v. stack: %v", stack.Spec.Name, version.Version, err, stack)
```

Comment:
I think the semver parser that we are using is pretty strict.  I think at the very least we need to say in the message that we require the version to have three parts (major.minor.patch).  I know that semver requires this, but it's one those things where some people think they are using semver even though it just has major.minor.  (another one is people who put the `v` on the front, like `v0.0.1`).  

Can we change the message to say "Stack %v %v spec.Versions[].Version must conform to semver major.minor.patch.  %v. stack: %v"  ?

Reply:
Dan looked up the message we get back from the semver parser and it's already doing a good job of telling the user what is wrong.